app/controllers/licence_finder_controller.rb

```diff
@@ -93,6 +93,18 @@ def browse_sector_child
     render "browse_sectors"
   end
 
+  def browse_licences
+    response.set_header("X-Robots-Tag", "noindex")
+
+    # There are < 500 licences, so it's fine to load all of this into memory
+    licences = Licence.order_by([%i[name asc]]).all
```

Comment:
Is this bit worth memoising?

Reply:
Do you mean caching beyond the scope of a single request? Or am I making multiple queries by accident here? I think that Mongoid will just make one query with an order_by which fetches all the data here.

Reply:
I'm going to merge it and see. It performed okay on my machine, so it should be find in prod where the mongo is bigger. I don't expect this page will get much traffic :)